### PR TITLE
Remove psake tool

### LINKED ISSUE #428: Review need for third party manifest file in our source tree
I am working on [our plan](https://docs.google.com/document/d/1fhbQWQaoIe-urizplgNtT4WxPRgoUDchg-VIBnAEqA0/edit?usp=sharing) (internal link) to rename `master` branches to `main`, as well as to remove this terminology wherever possible.

Why is [this file](https://github.com/ably/ably-dotnet/blob/d099e36bb49b4d9baa439092986f57d47decb975/tools/psake/psake.psd1) in our repository? Can it be refactored or replaced in order to remove references to "master"?

### PR DESCRIPTION
**Pull request by sacOO7**
- Removed psake tool and all relevant references.
- Updated readme for the same.
- Fixes #428

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -428,8 +429,9 @@ You can also view the [community reported Github issues](https://github.com/ably
 
 ## Building and Packaging
 
-The build scripts are written partly using `fake' and partly in powershell using PSake and need to be run on Windows with Visual Studio 2017 installed. Additionally nuget.exe and GitVersion.exe are required, these can be installed via [chocolatey](https://chocolatey.org)
+The build scripts are written using `fake' and need to be run on Windows with Visual Studio 2017 installed. Additionally nuget.exe and GitVersion.exe are required, these can be installed via [chocolatey](https://chocolatey.org)
```
Comment (marto83) [inline README.md:432]: We don't use Gitversion any more. You can remove that as well

Comment (sacOO7) [reply README.md:432]: Sure : )

**Comment by sacOO7** [comment]
Btw, @marto83 , I was going through the code, seems lots of unused code/tools/files, can be removed as a part of refactoring.

**Comment by sacOO7** [comment]
Would like to mention some of the files/tools as a part of refactoring. Can be cleaned as a part of new PR 
1. appveyor.yml
2. tools/powershell/FileSupport
3. tools/xunit
4. tools/7za.exe
5. tools/GetChildItemExtension.ps1
7. tools/curl.exe

@marto83 Can you recheck if these files/tools are needed.

**Comment by marto83** [comment]
@sacOO7 you can remove all the files you listed above

**Comment by sacOO7** [comment]
Also just a thought, regarding NuGet.exe, ILRepack.exe and other binaries ... can we automate installation using chocolaty during build. I am not sure, if targeted binaries will be able to run properly on different systems.

File: README.md (hunk 1 above)
Comment (sacOO7) [reply README.md]: @marto83 I have added a commit to update readme for the same. Can you check again : )

**Comment by marto83** [comment]
looks good 👍 
I'm happy for you to merge it to the main branch.

**Comment by sacOO7** [comment]
@QuintinWillison can we create a new issue for https://github.com/ably/ably-dotnet/pull/431#issuecomment-668086709 as a part of refactoring unused code 
I would also recommend creating one more issue for Fixing failing builds and tests on local and CI.

**Comment by QuintinWillison** [comment]
@sacOO7 please feel free to go ahead and create issues as you see fit. I'll review them as I see they come in. Leave them unassigned if you're not going to work on them in the immediate term. Thanks.

File: README.md [hunk 2]
```diff
@@ -428,8 +429,9 @@ You can also view the [community reported Github issues](https://github.com/ably
 
 ## Building and Packaging
 
-The build scripts are written partly using `fake' and partly in powershell using PSake and need to be run on Windows with Visual Studio 2017 installed. Additionally nuget.exe and GitVersion.exe are required, these can be installed via [chocolatey](https://chocolatey.org)
+The build scripts are written using `fake' and need to be run on Windows with Visual Studio 2017 installed. Fake and nuget.exe can be installed via [chocolatey](https://chocolatey.org)
```
Comment (QuintinWillison) [inline README.md:432]: I recognise that you've inherited this from what was there before but it looks like the formatting of the first instance of "fake" here is incorrect as it begins with a backtick but doesn't end in one. See [the rendered output for this branch](https://github.com/ably/ably-dotnet/blob/feature/428-remove-psake/README.md#building-and-packaging):

Screenshot 2020-08-21 at 09 18 15

File: README.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
 # ably-dotnet
+# ably-dotnet
```
Comment (QuintinWillison) [inline README.md:2]: You appear to have introduced a duplicate of the primary header!

**Review by marto83 — APPROVED** [review]
Looks good

**Review by QuintinWillison — APPROVED** [review]

**Pull request merged**